openapi.yml

```diff
@@ -107,5 +144,18 @@ components:
           $ref: "#/components/schemas/Address"
     OrderFull:
       allOf:
-        - $ref: "#/components/schemas/OrderNew"
+        - $ref: "#/components/schemas/OrderCreation"
```

Comment:
I know naming was discussed previously.

I'd slightly prefer the union type to be called Order (as I assume this is what most API clients will interact with by default and having the `Full` postfix seems a bit unspecific and verbose) and OrderMetadata over OrderExtra (which to me seems like an extra order type). Not sure if those names will actually show up in the API or codebase if not it's probably not worth fighting over them. 

I really like OrderCreation for the part that contains all submission information.